modules/test/playwright/tests/journal-web/journalArticle.spec.ts

```diff
@@ -970,16 +970,24 @@ scheduleTest(
 		await journalEditArticlePage.goto({siteUrl: site.friendlyUrlPath});
 
 		const articleTitle = getRandomString();
-		const scheduleDate = '9987-11-26 13:00';
+		const expirationDate = '01/01/9999';
+		const publishDate = '9987-11-26 13:00';
+		const reviewDate = '01/01/9999';
```

Comment:
It is just curious why this `expirationDate` and `reviewDate` have different format than `publishDate`... ;-)

Reply:
I don't know why exactly, but the publish date and the expiration/review dates pickers are different (see screenshots)   
 

![](https://github.com/liferay-content-management/liferay-portal/assets/91208451/6d86851f-a06a-4441-9bb5-ade7e7bc492c)

![](https://github.com/liferay-content-management/liferay-portal/assets/91208451/07b59f5c-4db9-4cc7-8395-e7e29107ef62)